app/models/user.rb

```diff
@@ -19,6 +19,10 @@ def admin?
     role == 'admin'
   end
 
+  def owner_of?(item)
+    admin? || id == item.user_id
```

Comment:
I think we should rename this method so that it's more clear what we're doing. An admin isn't really an owner, per se, and this leaves room for confusion. 

Could we try something like  can_modify or can_manage ?